Public/Src/FrontEnd/UnitTests/Script.Ast/Incrementality/TestSerializationCompleteness.cs

```diff
@@ -526,6 +526,12 @@ public void TestConditionalExpression()
             ConditionalExpression node = new ConditionalExpression(GetExpression1(), GetExpression2(), GetExpression1(), DefaultLineInfo);
             CheckSerializationRoundTrip(node);
         }
+        
+        [Fact]
+        public void TestSwitchExpression()

```

Comment:
>TestSwitchExpression [](start = 20, length = 20)

Now that I see a second TODO, maybe you are not planning to write this for this push? Anyway, leaving this comment just in case

Reply:
nope, got too excited, I used to review the PR in the Azure DevOps web UI before creating it, but the GitHub diff UI is too painfull to use, so I have to learn to start reviewing locally before creating the PR....

---
In reply to: [274111021](https://github.com/Microsoft/BuildXL/pull/119#discussion_r274111021) [](ancestors = 274111021)